libnavigation-util/src/main/java/com/mapbox/navigation/utils/timer/MapboxTimer.kt

```diff
@@ -1,35 +1,34 @@
 package com.mapbox.navigation.utils.timer
 
 import com.mapbox.navigation.utils.thread.ThreadController
+import java.util.concurrent.TimeUnit
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.cancelChildren
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.isActive
 import kotlinx.coroutines.launch
 
 /**
- * Schedules a delay of [restartAfter] milliseconds and then restarts.
+ * Schedules a delay of [routeRefreshInterval] milliseconds and then restarts.
  *
  * @param restartAfter Time delay until the timer should restart.
- * @param executeLambda lambda function that is to be executed after [restartAfter] milliseconds.
+ * @param executeLambda lambda function that is to be executed after [routeRefreshInterval] milliseconds.
  */
-class MapboxTimer(private val restartAfter: Long, private val executeLambda: () -> Unit) {
-    private val mainControllerJobScope = ThreadController.getMainScopeAndRootJob()
+class MapboxTimer {
+    private val jobControl = ThreadController.getMainScopeAndRootJob()
 
-    private val timerJob: Job by lazy {
-        mainControllerJobScope.scope.launch {
+    var routeRefreshInterval = TimeUnit.MINUTES.toMillis(1)
```

Comment:
If this is a "generic" timer, shouldn't we use a more abstract naming? 🤔 Also in this particular case it's route refresh is confusing because we already have the concept (feature) of route refresh 😅 The same thing happens to `startRouteRefresh` 👇 What do you think?

Reply:
It's still a MapboxTimer, not a EveryoneTimer. This class is only used to update traffic routes, tend to make things more specific at the beginning. When it's used in more places it could become more abstract.

But actually, this topic is out of scope of the refactor I'm doing so will go back to the original name